bash_scripts/tests.sh

```diff
@@ -46,7 +46,7 @@ apt-get -y install git
 apt-get install swig -y
 
 # Install depedencies
-pip install .[tf,envs,reverb,testing_formatting,record_episode]
+pip install .[jax,envs,reverb,testing_formatting,record_episode]
```

Comment:
and if all the tests are being run now (including tf ones), why is the tf dependency removed?

Reply:
That is misleading. One would think that JAX and TF are separate but I guess it is a temporary solution until we deprecate JAX